py/requirements-win32.txt

```diff
@@ -2,5 +2,6 @@
 ## This is a subset of the dependencies defined in py/requirements.txt. These dependencies can't be installed via the OS
 ## package manager.
 ##
+urllib3>=1.24.1
```

Comment:
Maybe it shall be bumped up in both requirements.txt and this one?

Reply:
I honestly have no idea, this is just a stab in the wild. I have very little insight in Python deps...